wp-content/themes/Belladonna/page.php

```diff
@@ -13,49 +13,108 @@
         <img src="<?php echo $header_image[0]; ?>" class="header_image" />
     <?php endif; ?>
 
-    <div class="main_section_wrapper section_<?php the_ID(); ?>">
+    <div class="main_section_wrapper">
         <h1 class="page-title <?= $page_color ?>"><?php the_title(); ?></h1>

```

Comment:
for `<? $page_color ?>` do 
`<?= 'u-color--' . strtolower($page_color); ?>` and same for the background color.

Reply:
Please don't use short form php like this. The main technical reason is portability, you can never be sure that the short form tags will work for every given setup, as they can be turned off (lookup the `short_open_tag` directive). But you can always be absolutely certain that the long-form will work everywhere.